lib/pbxProject.js

```diff
@@ -1546,7 +1546,7 @@ function pbxShellScriptBuildPhaseObj(obj, options, phaseName) {
     obj.inputPaths = options.inputPaths || [];
     obj.outputPaths = options.outputPaths || [];
     obj.shellPath = options.shellPath;
-    obj.shellScript = '"' + options.shellScript.replace(/"/g, '\\"') + '"';
+    obj.shellScript = '"' + options.shellScript.replace(/\\/g, '\\\\') + '"';
```

Comment:
Doing this you remove the escaping of `"`, I should have both, right?

Reply:
https://github.com/apache/cordova-node-xcode/commit/3d5657ef3bbd5dd281def559fd85cee667637b9b#diff-35bc421794193e73d972308b4f2f3598

Reply:
https://github.com/apache/cordova-node-xcode/commit/3d5657ef3bbd5dd281def559fd85cee667637b9b#diff-35bc421794193e73d972308b4f2f3598